From e5690758b47f53c662c50ae1ed39164f1c21ab7f Mon Sep 17 00:00:00 2001
From: qianmenx <qianx.meng@intel.com>
Date: Thu, 16 May 2019 11:22:03 +0800
Subject: [PATCH 2317/2328] media: intel-ipu4: fix frame counter(x3a) can't
 work issue

Message for Open Source:
fix frame counter(x3a) can't work issue

Change-Id: I7c5b694a1acc44248959be2ed8b16aa390885ab2
Signed-off-by: qianmenx <qianx.meng@intel.com>
(cherry picked from commit 57aca479a7ce83eb8f3aff45e131f50101e0ffb1)
Signed-off-by: lijuncax <lijunx.cai@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
index 486c9ca..34dab57 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
@@ -529,6 +529,7 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG[] = {
 	{ 0x384d, CRL_REG_LEN_08BIT, 0x14 },
 	{ 0x460a, CRL_REG_LEN_08BIT, 0x0e },
 	/* embedded data */
+	{ 0x484c, CRL_REG_LEN_08BIT, 0x02 },
 	{ 0x3208, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x483e, CRL_REG_LEN_08BIT, 0x02 },/* frame counter */
 	{ 0x3501, CRL_REG_LEN_08BIT, 0x02 },/* DCG exp */
-- 
2.7.4

